src/electron/views/automated-checks/components/command-bar.scss

```diff
@@ -12,4 +12,12 @@
     font-size: 14px;
     font-weight: normal;
     border-bottom: 1px solid $neutral-alpha-8;
+    padding: 0px 16px;
+
+    #rescan-button {
```

Comment:
Is it necessary to use an id rather than a class for this? If there's a specificity issue you might be able to work around it by setting the classname you need as an `iconProps` property instead

Reply:
Good call. I updated and put the `className` on the `iconProps` as you suggested